UnicornRender/video/include/unicorn/video/vulkan/VkMesh.hpp

```diff
@@ -81,6 +81,9 @@ class VkMesh
     /** @brief Returns constant reference to unicorn::Mesh */
     Mesh const& GetMesh() const;
 
+    /** @brief Returns sprite coordinates */
+    glm::vec4 GetSpriteCoordinates() const;
```

Comment:
I think the return format deserves some description.

Reply:
Whoa, this line must not even exist)